### ISSUE TITLE
cannot navigate w arrow keys in terminal accessibility buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#172222]
inserting adjacent is the problem - needs to be `this._terminal.element?.appendChild(this._accessibilityTreeRoot);`

specifically https://github.com/xtermjs/xterm.js/pull/4382#discussion_r1084648740

https://user-images.githubusercontent.com/29464607/214367652-d14b2f47-9f83-410e-bfb6-96f99f1f5a19.mov

**Comment by meganrogge** [comment]
this is still a problem today in oss off of main


https://user-images.githubusercontent.com/29464607/214602338-908d6356-c843-432b-8916-14cdcd91981b.mov

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/issues/172394

**Comment by rzhao271** [comment]
LGTM on Windows
